netdata-installer.sh

```diff
@@ -394,6 +388,57 @@ fi
 # replace multiple spaces with a single space
 NETDATA_CONFIGURE_OPTIONS="${NETDATA_CONFIGURE_OPTIONS//  / }"
 
+NEED_PROTOBUF=true
+
+if [ -n "${USE_SYSTEM_PROTOBUF}" ]; then
+  NEED_PROTOBUF=false
+fi
+
+if [ -n "${NETDATA_DISABLE_CLOUD}" ] && [ -n "${NETDATA_DISABLE_PROMETHEUS}" ]; then
+  NEED_PROTOBUF=false
+fi
+
+if ${NEED_PROTOBUF} && [ "$(uname -s)" = "Linux" ] && [ -f /proc/meminfo ]; then
+  mega="$((1000 * 1000))"
```

Comment:
Shouldn't we use base 2 here?

> 1024 * 1024

Reply:
SI units were picked in this case because they get us as close as possible to the actual values we care about without using ‘weird’ multipliers.

Reply:
the number you see using `free` are not in si by default. MemTotal (/proc/meminfo) is also not si.